dependencies.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+    compile("com.github.GTNewHorizons:GT5-Unofficial:unify-build-script-SNAPSHOT:dev") {
```

Comment:
Why don't use the default branch? As soon as the GT5U PR is merged, this has to be changed again

Reply:
It reduces the number of dependencies by a lot and increases startup time in dev. I'll leave this unresolved so I won't forget it.